pageserver/src/http/openapi_spec.yml

```diff
@@ -363,11 +363,27 @@ paths:
         * MUST NOT ASSUME that the request has been lost, based on the observation
           that a subsequent tenant status request returns 404. The request may
           still be in flight. It must be retried.
+
+        The client SHOULD supply a `TenantConfig` for the tenant in the request body.
+        Settings specified in the config override the pageserver's defaults.
+        It is guaranteed that the config settings are applied before the pageserver
+        starts operating on the tenant. E.g., if the config specifies a specific
+        PITR interval for a tenant, then that setting will be in effect before the
+        pageserver starts the garbage collection loop. This enables a client to
+        guarantee a specific PITR setting across detach/attach cycles.
+        The pageserver will reject the request if it cannot parse the config, or
+        if there are any unknown fields in it.
+        If the client does not supply a config, the pageserver will use its defaults.
+      requestBody:
+        required: false
```

Comment:
If the body is empty json, then it is equivalent to empty body?

Reply:
Yeah, I'm pondering about that. Basically, I wanted to make this change backwards-compatible. Now I'm thinking we could just make it breaking and require a body. Because, the API isn't in use yet.

What do you think?
Should we just require a valid body, i.e., at a minimum, `{}`? As I said, it's technically a breaking change but practically the API isn't used yet.

The only usages are:
* [the ad-hoc guide in the wiki](https://github.com/neondatabase/cloud/wiki/Cloud:-Ad-hoc-tenant-relocation#11-disable-maintenance-mode)
* the pageserver python tests
* the cloud e2e tests

Reply:
I am good with optional body, have never liked possible empty bodies as required :)

Reply:
Decision made end of last week: for now we'll make this backwards-compatible so that we don't have to forward-break `cloud` to satisfy e2e tests here.

But we'll make a well-formed body mandatory later https://github.com/neondatabase/neon/issues/4282